scripts/steps.yml

```diff
@@ -0,0 +1,40 @@
+steps:
+
+##
+# Pre-reqs
+##
+- task: UseDotNet@2
+  displayName: 'Use .NET Core sdk'
+  inputs:
+    packageType: sdk
+    version: 2.2.x
+
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.6'
+    architecture: 'x64'
+  displayName: 'Use Python 3.6'
+
+- script: pip install setuptools wheel pytest jupyter
+  displayName: 'Install Python tools'
+
+- powershell: ./install-iqsharp.ps1
+  displayName: "Installing IQ#"
+  workingDirectory: $(System.DefaultWorkingDirectory)/scripts
+
+##
+# Build, test & pack
```

Comment:
Is there a way to do the building and (most importantly!) the testing locally using the same tools? Could you please document this?

Reply:
Yes, it's very easy: just run the `scripts\build.ps1` and then run the `scripts\test.ps1` from powershell.
You will need to have installed the dependencies (powershell, dotnet, juypter).
I don't have time to document it right now, though.

Reply:
Documented a bit in https://github.com/microsoft/QuantumKatas/pull/159.